src/Platform/Microsoft.Testing.Platform/Configurations/ConfigurationManager.cs

```diff
@@ -16,7 +16,7 @@ internal sealed class ConfigurationManager : IConfigurationManager
 
     public void AddConfigurationSource(Func<IConfigurationSource> source) => _configurationSources.Add(source);
 
-    internal async Task<IConfiguration> BuildAsync(IServiceProvider serviceProvider, FileLoggerProvider? syncFileLoggerProvider)
+    internal async Task<IConfiguration> BuildAsync(IServiceProvider serviceProvider, ILoggerProvider? syncFileLoggerProvider)
```

Comment:
@MarcoRossignoli Please review this change. From the field name, I think this is a case where we DO NEED to have a file logger instance.

Reply:
Yep we use it to log during the infra building, so we call create and write in a sync way to the `FileLogger` on purpose.

Reply:
Maybe the easiest is to introduce a `IFileLoggerProvider` and replace this use by that. This is a specialized interface and can be easily mocked.

Reply:
Fixed, thanks for the suggestion.